backend/src/v5/utils/responseCodes.js

```diff
@@ -0,0 +1,93 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+const { toCamelCase, toSnakeCase } = require('./helper/strings');
+
+const ResponseCodes = {};
+
+ResponseCodes.template = {
```

Comment:
I think  `.template` to `.templates` would reflect a bit better this because in the rest of the system is being used as a collection of templates or response messages.